samples/csharp/MicrosoftEduGraphSamples/Workflows/AssignmentWorkflow.cs

```diff
@@ -214,5 +214,61 @@ public async Task AssignmentResourceAsync(bool appOnly = false)
+        /// </summary>
+        /// <param name></param> 
+        public async Task GetRecentlyModifiedSubmissionsGetResponseAsync()
+        {
+            try
+            {
+                string assignmentId = string.Empty;
+                string submissionId = string.Empty;
+                int numberOfAssignments = 10;
+                List<string> assignmentIds = new List<string>();
+
+                // Get a Graph client using delegated permissions
+                var graphClientTeacherRole = GraphClient.GetDelegateClient(_config["tenantId"], _config["appId"], _config["teacherAccount"], _config["teacherPassword"]);
+                var graphClientStudentRole = GraphClient.GetDelegateClient(_config["tenantId"], _config["appId"], _config["studentAccount"], _config["studentPassword"]);
+
+                for (int i = 0; i < numberOfAssignments; i++)
+                {
+                    // Create assignment to verify inactive state
+                    var assignmentInactive = await Assignment.CreateSampleAssignmentAsync(graphClientTeacherRole, _config["classId"]);
+
+                    //Store Assignment ID
+                    assignmentId = assignmentInactive.Id;
+
+                    Console.WriteLine($"Assignment {i + 1} created successfully: ID = {assignmentInactive.Id}, Status = {assignmentInactive.Status}");
+
+                    // Publishing each Assignment
+                    assignmentInactive = await GlobalMethods.PublishAssignmentsAsync(graphClientTeacherRole, assignmentInactive.Id);
+                    Console.WriteLine($"Assignment {i + 1} published successfully: ID = {assignmentInactive.Id}, Status = {assignmentInactive.Status}");
+
+                    // Get the student submission using Expand
+                    var submissions = await Submission.GetSubmissionsWithExpandAsync(graphClientStudentRole, _config["classId"], assignmentId, "outcomes");
+                    if (submissions.Value.Count > 0)
+                    {
+                        submissionId = submissions.Value[0].Id;
+                        Console.WriteLine($"Submission {submissionId} found for {_config["studentAccount"]}");
+                    }
+                    else
+                    {
+                        throw new Exception($"No submission found for {_config["studentAccount"]} in {assignmentId} for class {_config["classId"]}");
+                    }
+
+                    // Student submits their submission
+                    var submission = await Submission.SubmitAsync(graphClientStudentRole, _config["classId"], assignmentId, submissionId);
+                    Console.WriteLine($"Submission {submission.Id} in state {submission.Status}");
+
+                }
+            }
+            catch (Exception ex)
+            {
+                Console.WriteLine($"GetRecentlyModifiedSubmissionsGetResponseAsync: {ex.ToString()}");
```

Comment:
I don't see a call to the GetRecentlyModifiedSubmissions endpoint

Reply:
@v-rmanda
your sample is missing all tests to the new endpoint